docs/experiments-msmarco-v2.md

```diff
@@ -0,0 +1,61 @@
+# Anserini: Guide to Working with the MS MARCO V2 Collections
```

Comment:
Maybe add a small description of what is different between v1 and v2 collections?

Reply:
Let me add that after v2 is officially release?

Reply:
SG!